system/Debug/Toolbar/Collectors/Database.php

```diff
@@ -138,59 +138,12 @@ protected function formatTimelineData(): array
 	 */
 	public function display(): array
 	{
-		// Key words we want bolded
-		$highlight = [
-			'SELECT',
-			'DISTINCT',
-			'FROM',
-			'WHERE',
-			'AND',
-			'LEFT&nbsp;JOIN',
-			'RIGHT&nbsp;JOIN',
-			'JOIN',
-			'ORDER&nbsp;BY',
-			'GROUP&nbsp;BY',
-			'LIMIT',
-			'INSERT',
-			'INTO',
-			'VALUES',
-			'UPDATE',
-			'OR&nbsp;',
-			'HAVING',
-			'OFFSET',
-			'NOT&nbsp;IN',
-			'IN',
-			'LIKE',
-			'NOT&nbsp;LIKE',
-			'COUNT',
-			'MAX',
-			'MIN',
-			'ON',
-			'AS',
-			'AVG',
-			'SUM',
-			'(',
-			')',
-		];
-
-		$data = [
-			'queries' => [],
-		];
-
-		foreach (static::$queries as $query)
-		{
-			$sql = $query->getQuery();
-
-			foreach ($highlight as $term)
-			{
-				$sql = str_replace($term, "<strong>{$term}</strong>", $sql);
-			}
-
-			$data['queries'][] = [
-				'duration' => ($query->getDuration(5) * 1000) . ' ms',
-				'sql'      => $sql,
+		$data['queries'] = array_map(function ($query) {
```

Comment:
For IDE intellisense
```suggestion
		$data['queries'] = array_map(function (Query $query) {
```

Reply:
Added 

I'm using PHPStorm and it already handles it properly as I changed 

```
	 * @var Query[]
	 */
	protected static $queries = [];
```

So I missed adding it in closure definition